src/combinatorics/simple_graph/ends/defs.lean

```diff
@@ -6,6 +6,7 @@ Authors: Anand Rao, Rémi Bottinelli
 import category_theory.mittag_leffler
 import combinatorics.simple_graph.connectivity
 import data.set_like.basic
+import data.finite
```

Comment:
I'm surprised you need this import.
```suggestion
```

Reply:
It seems I need it for `finite.of_injective_finite_range` (line 242), but i can actually remove `data.set_like.basic`

Reply:
Wait, that's a default file. Can you replace it by the imports you're actually using?

Reply:
TIL default files :)